src/components/TokenLocking/styles.module.css

```diff
@@ -194,4 +194,68 @@
 
 .diamondImage {
   text-align: center;
+}
+
+.milesReceipt {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 1000px;
+  height: 450px;
+  display: flex;
+  overflow: hidden;
+}
+
+.leftReceipt, .rightReceipt {
+  background-color: #121312;
+  border-radius: 20px;
+  padding: 40px;
+}
+
+.leftReceipt {
+  position: relative;
+  flex-grow: 1;
+}
+
+.leftReceipt:after {
+  content: "";
+  border-right: 1px dashed white;
```

Comment:
Nit:
This dashed border does not match the designs. If we have time later we could play around here with a background image and a transparent border to control the dashes.